src/views/Settings.vue

```diff
@@ -140,6 +168,12 @@ export default {
       this.$store.dispatch("cleanAll");
       this.localApiToken = this.$store.state.apiToken;
       this.localApiEndpoint = this.$store.state.apiEndpoint;
+      this.localDarkMode = this.$store.state.darkMode;
+    }
+  },
+  watch: {
+    localDarkMode(toggle) {
+      this.$store.dispatch("updateTheme", toggle);
```

Comment:
Remove this useless watcher and dispatch in the setter has described above, WDYT?